libs/pika/async_mpi/include/pika/async_mpi/transform_mpi.hpp

```diff
@@ -12,6 +12,7 @@
 #include <pika/config.hpp>
 #include <pika/assert.hpp>
 #include <pika/async_mpi/mpi_polling.hpp>
+#include <pika/async_mpi/mpi_throttling.hpp>
```

Comment:
This header doesn't exist by the way... All functionality is in `mpi_polling.hpp`, right?